coalib/parsing/InvalidFilterException.py

```diff
@@ -1,2 +1,19 @@
-class InvalidFilterException(Exception):
-    pass
+class InvalidFilterException(BaseException):
+
+    def __init__(self, filter_name, available_filters):
+        self.filter = filter_name
+        self.available_filters = list(available_filters.split(', '))
+        if self.filter is None and self.available_filters is None:
+            message = 'Invalid Filter'
+
+        if self.filter is None and self.available_filters is not None:
```

Comment:
`self.filter` is never going to be `None`. Because argparser will make sure that there will be at least one filter available.